vendor/extensions/events/app/controllers/refinery/events/events_controller.rb

```diff
@@ -8,11 +8,10 @@ class EventsController < ::ApplicationController
       def index
         # you can use meta fields from your model instead (e.g. browser_title)
         # by swapping @page for @event in the line below:
-
-        @upcoming_events = ::Refinery::Events::Event.where(start: DateTime.now..DateTime.now.at_end_of_month)
+        @upcoming_events = ::Refinery::Events::Event.where(start: DateTime.now..DateTime.now.at_end_of_month).sort {|a,b| a.start <=> b.start }
```

Comment:
What about `.order` to send an order request to the DB? https://guides.rubyonrails.org/active_record_querying.html#ordering

Reply:
Thanks, Matt I have refactored using `.order`